Sources/Sentry/SentryExtraPackages.m

```diff
@@ -0,0 +1,47 @@
+#import "SentryExtraPackages.h"
+#import "SentryMeta.h"
+
+static NSSet<NSDictionary<NSString *, NSString *> *> *extraPackages;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation SentryExtraPackages
+
++ (void)addPackageName:(NSString *)name version:(NSString *)version
+{
+    if (name == nil || version == nil) {
+        return;
+    }
+
+    @synchronized(extraPackages) {
+        NSDictionary<NSString *, NSString *> *newPackage =
+            @{ @"name" : name, @"version" : version };
+        if (extraPackages == nil) {
+            extraPackages = [[NSSet alloc] initWithObjects:newPackage, nil];
+        } else {
```

Comment:
`m`: Instead of checking for nil here, you can add a static init method, as we have it here:

https://github.com/getsentry/sentry-cocoa/blob/716a5b0ea1d64474b2285f5799a81ff0faac918f/Sources/Sentry/SentrySDK.m#L71-L80